timemachine/fe/free_energy.py

```diff
@@ -67,12 +69,15 @@ class PairBarPlots:
     overlap_detail_png: bytes
 
 
+Frames = TypeVar("Frames")
```

Comment:
Not for this PR: We seem to be defining the same types in multiple places, should try to put them in one place to ensure consistency